### ISSUE TITLE
How to set batch_size,port,character paramter?

### ISSUE THREAD

**Issue by ericwang321** [mikealfare/bcp#13]
**Thank you very much for your code.**

In the  python3 environment:

**How to set  like a command line?**
```
-b batch_size
-e errorfile
-c Causes the bcp utility to use character data.*
Sql port (such as 14331)
```
thanks

**Comment by mikealfare** [comment]
Thanks for your feedback. The short answer is that you cannot set those currently.

The first three are hard-coded (-b 10000, -e writes to your %userprofile% on Windows, and -c is passed in). The fourth is missing.

I think adding -b, -c, and port are easy enough. For the error file, are you looking for a flag, for whether to write to an error file? Or are you looking to pass in a path to use instead of the default? Or both?

**Comment by ericwang321** [comment]
I want to set the following three functions. 
```

-b batch_size (can set from 1 )
-c Causes the bcp utility to use character data.*
-p Sql port (such as 14331)...
```

As for the error file, the default settings can be fine,write to an error file.
if in the ubuntu environment, which folder is the log?

**Comment by mikealfare** [comment]
I looked at how I could add those. -b and -c would be set on the calls (BCP.load() or BCP.dump()). -p would be added to the Connection.

When I defaulted the directory for the error file, I was only thinking about Windows. It writes to %userprofile%. That's an environment variable in Windows, but I don't think it is in Ubuntu. I've been meaning to update that anyway. I'll see if there's an easy way to write to the user's home directory in either environment. I feel like there is. And the log file is the same deal. It writes to %userprofile%/logs.
